docs/shared/getting-started/installation.md

```diff
@@ -7,144 +7,64 @@ description: Learn how to install Nx in a new workspace, add it to an existing r
 
 Create a new Nx workspace using the following command:
 
+To install Nx on your machine, choose one of the following methods based on your operating system and package manager. You can also use `npx` to run Nx without installing it globally.
+
 {% tabs %}
 {% tab label="npm" %}
 
 ```shell
-npx create-nx-workspace
-```
-
-{% /tab %}
-{% tab label="yarn" %}
-
-```shell
-npx create-nx-workspace --pm yarn
-```
-
-{% /tab %}
-{% tab label="pnpm" %}
-
-```shell
-npx create-nx-workspace --pm pnpm
-```
-
-{% /tab %}
-{% tab label="Bun" %}
-
-```shell
-bunx create-nx-workspace --pm bun
+# You can also use yarn, pnpm, or bun
```

Comment:
We can move this outside the `shell` block, just as a normal text within the tab. Otherwise the rendering looks weird

![Screenshot 2025-06-10 at 11 21 06](https://github.com/user-attachments/assets/f3ca5d5a-43b4-4baa-9ed5-d0c3b473c640)